lucupy/sky/brightness.py

```diff
@@ -159,37 +144,28 @@ def calculate_sky_brightness_qpt(moon_phase_angle: Quantity,
                                  verbose: bool = False) -> npt.NDArray[float]:
     """
     Calculate sky brightness based on formulas from Krisciunas & Schaefer 1991
-    Bryan Miller
-    November 5, 2004
-    June 1, 2015 - added cc parameter while testing cloud scattering corrections
-
-    Matt Bonnyman
-    converted from IDL to Python May 23, 2018
 
-    Parameters
-    ----------
-    moon_phase_angle : '~astropy.units.Quantity'
-        Moon phase angle at solar midnight in degrees
+    Changelog:
 
-    target_moon_angdist : '~astropy.units.Quantity'
-        Numpy array of angular distances between target and moon
+        Bryan Miller
+        November 5, 2004
+        June 1, 2015 - added cc parameter while testing cloud scattering corrections
 
-    moon_zenith_distang : '~astropy.units.Quantity'
-        Numpy array of Moon zenith distance angles
+        Matt Bonnyman
+        May 23, 2018
+        converted from IDL to Python
```

Comment:
We should probably get rid of this. I'll add Matt Bonnyman to the list of contributors, but I don't think we need this cruft anymore.